src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -64,15 +79,32 @@ def scan_for_free_port(starting_port: int, step: int = 200):
     raise RuntimeError("could not allocate free port for proxy")
 
 
-def run_command(command, shell=False, raise_errors=True, env=None, cwd=None, run_in_background=False):
+def _subprocess_popen(command, shell=False, env=None, cwd=None, stderr=subprocess.STDOUT):
+    subprocess.Popen(command, shell, env, cwd, stderr)
+
+
+def _subprocess_run(command, shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=None,
+                    universal_newlines=None, cwd=None, check=None):
+    process = subprocess.run(
+        command, shell=shell, stdout=stdout, stderr=stderr, env=env, universal_newlines=universal_newlines,
+        cwd=cwd, check=check)
+    return process
+
+
+def run_command(command, shell=False, raise_errors=True, env=None, cwd=None, run_in_background=False,
+                stderr=subprocess.PIPE, stdout=subprocess.PIPE, universal_newlines=True, check=False):
+
+    if global_variables().remote_shell_address:
+        remote_shell = _initialize_remote_shell()
+        return remote_shell.run_command(command, raise_errors=raise_errors, background=run_in_background)
+
```

Comment:
I think we should seperate the run_command to to functions:
run_command
run_command _background ....
the reason is that subprocess.popen and subprocess.run use different params and it causes mess

what do you think ? who is running today with run_in_background=True ?

Reply:
I see no usages for `run_in_background=True` could be worth to get rid of it